src/components/menu/ComingSoon/ComingSoon.js

```diff
@@ -0,0 +1,44 @@
+import './GoogleBrandFormatting.css'
+
+
+function ComingSoon ()
+{
+    return ( 
+        <div className='center'>  
+<h1 className='h1-green'> <span class = "h1-blue">
+C
+    </span>
+    <span class = "h1-red">
+o
+    </span>
```

Comment:
Let's do 1 line per letter
```suggestion
    <span class = "h1-red">o</span>
```

Reply:
Yes! This is cleaner! Will focus on that.

Reply:
Completed.